src/System.Memory/ref/System.Memory.cs

```diff
@@ -11,7 +11,7 @@ namespace System
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
     public partial struct ReadOnlySpan<T>
     {
-        public static readonly System.ReadOnlySpan<T> Empty;
+        public static System.ReadOnlySpan<T> Empty { get; }
```

Comment:
In all of our refs, properties with getters have those getters doing "throw null;".  As it's written, the compiler will generate a field to back this and a getter that returns it.